src/app/core/basic-datatypes/date-with-age/date-with-age.datatype.ts

```diff
@@ -7,7 +7,9 @@ import { DateWithAge } from "./dateWithAge";
  */
 @Injectable()
 export class DateWithAgeDatatype extends DateOnlyDatatype {
-  static dataType = "date-with-age";
+  static override dataType = "date-with-age";
+  static override label: string = $localize`:datatype-label:date with age`;
```

Comment:
it could be useful for something like an admission date and the years of being in the program, also. But you might be right that it is easier for people to understand "date of birth" - possibly even when making the mental jump of using a "date of birth" for the other use cases.

```suggestion
  static override label: string = $localize`:datatype-label:date of birth (date + age)`;
```